defaults/main.yml

```diff
@@ -10,6 +10,7 @@ marathon_apt_repo: "deb http://repos.mesosphere.com/{{ansible_distribution|lower
 marathon_yum_package: "marathon-{{ marathon_version }}"
 mesosphere_yum_repo: "http://repos.mesosphere.com/el/{{ os_version_major }}/noarch/RPMS/{{ mesosphere_releases[os_version_major] }}"
 
+marathon_framework_name: marathon
```

Comment:
Could you please use the default value used by Marathon, which is `marathon-VERSION` where version is version of marathon of course.